src/main/scala/mesosphere/marathon/state/GroupManager.scala

```diff
@@ -155,7 +159,9 @@ class GroupManager @Singleton @Inject() (
       from <- rootGroup()
       (toUnversioned, resolve) <- resolveStoreUrls(assignDynamicServicePorts(from, change(from)))
       to = GroupVersioningUtil.updateVersionInfoForChangedApps(version, from, toUnversioned)
-      _ = BeanValidation.requireValid(ModelValidation.checkGroup(to, "", PathId.empty, config.maxApps.get))
+      _ = {
+        validateOrThrow(V2Group(to))(validator = V2Group.v2GroupWithConfigValidator(config.maxApps.get))
+      }
```

Comment:
1) Remove the curly braces.
2) Can we create a validator in Group that delegates to V2Group (until we remove it)
```scala
validateOrThrow(to)(Group.groupValidator(config.maxApps())
```
```